test/test_operators.py

```diff
@@ -1,6 +1,7 @@
 import unittest
-from test.util import TVMTest
+from util import TVMTest
```

Comment:
pls don't change this :)

try using `python -m unittest test.test_operators.TestOperators.test_max_pool2d`

Reply:
oops didn't mean to commit this, it's only for my local testing, I will remove